.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
Thanks for adding this file! We might also want to be more specific if we're adding this. I'm using `16.20.2` specifically.